content/docs/user-guide/troubleshooting.md

```diff
@@ -91,3 +91,17 @@ using:
 ```dvc
 $ dvc checkout --relink
 ```
+
+## HTTP Git authentication is not supported {#git-auth}
+
+[Experiment sharing](/doc/user-guide/experiment-management/sharing-experiments)
+commands accept a `git_remote` argument. In order to access the Git remote,
+you may need to authenticate for _reading_ (`dvc exp list`,
+`dvc exp pull`) or _writing_ (`dvc exp push`) permissions.
+
+This means that unless the Git server allows unauthenticated HTTP
+read/write, you should use an [SSH Git URL] when pushing experiments,
+and when listing or pulling experiments from private repositories.
```

Comment:
I think it should be simpler like this? The "unless" part already implies the cases where HTTP is OK.
```suggestion
This means that unless the Git server allows unauthenticated HTTP
write/read, you should use an [SSH Git URL] when pushing, listing,
or pushing experiments.
```